src/pages/snap/{Snap.location}/{Snap.slug}.tsx

```diff
@@ -120,6 +121,11 @@ const SnapPage: FunctionComponent<SnapPageProps> = ({ data }) => {
               }
             />
           )}
+          <Box
+            display={{ base: 'none', md: 'flex', lg: 'none' }}
+            flexBasis="100%"
+            height={0}
+          ></Box>
```

Comment:
Maybe add a comment too why we need an empty box?

```suggestion
          <Box
            display={{ base: 'none', md: 'flex', lg: 'none' }}
            flexBasis="100%"
            height={0}
          />
```

Reply:
Updated.